DSCResources/MSFT_xDnsRecord/MSFT_xDnsRecord.schema.mof

```diff
@@ -5,6 +5,7 @@ class MSFT_xDnsRecord : OMI_BaseResource
     [Key] string Zone;
     [Required, ValueMap{"ARecord","CName"}, Values{"ARecord","CName"}] string Type;
     [Key] string Target;
+    [write] string DnsServer;

```

Comment:
Can you please capitalize the 'W' in 'write' here?
It doesn't make a difference to the functionality.
Just for style consistency.